### PR TITLE
ZEN-17380 ZEP can't find plugins in link installed zenpacks

### PR DESCRIPTION
**Pull request by jpeacock-zenoss**

### CODE REVIEW

**Comment by epowell** [comment]
ok to test

File: src/main/java/org/zenoss/utils/ZenPacks.java [hunk 1]
```diff
@@ -30,54 +30,86 @@
...
+    static {
+        zenpackDirectories = new HashSet<String>();
+        try {
+            zenpackDirectories.add(Zenoss.zenPath("ZenPacks"));
+        } catch(Exception e) {
+            // Assume the exception was due to $ZENHOME being undefined and assume ZENHOME is /opt/zenoss.
+            zenpackDirectories.add("/opt/zenoss/ZenPacks");
+        }
+        zenpackDirectories.add("/var/zenoss/ZenPacks");
+    }
```
Comment (epowell) [inline src/main/java/org/zenoss/utils/ZenPacks.java]: We can do this in a Spring initialization call to make this configurable in a .properties file.

File: src/main/java/org/zenoss/utils/ZenPacks.java [hunk 2]
```diff
@@ -30,54 +30,86 @@
...
 
+    private static Collection<String> zenpackDirectories = null;
+
+    // Statically define the set of directories where easy-install.pth files can be as
+    // $ZENHOME/ZenPacks and /var/zenoss/ZenPacks.
+    static {
+        zenpackDirectories = new HashSet<String>();
+        try {
+            zenpackDirectories.add(Zenoss.zenPath("ZenPacks"));
+        } catch(Exception e) {
```
Comment (epowell) [inline src/main/java/org/zenoss/utils/ZenPacks.java]: Is it the `zenPath` call that throws here? or the add?

Comment (jpeacock-zenoss) [inline src/main/java/org/zenoss/utils/ZenPacks.java]: zenPath throws it.  IIRC, it's thrown when $ZENHOME is undefined.

**Comment by epowell** [comment]
Builds fine on my box - Jenkins had a problem with git interaction.

**Pull request merged**